engine/src/main/java/org/terasology/engine/module/ModuleManagerImpl.java

```diff
@@ -128,7 +125,7 @@ public ModuleManagerImpl(Config config, TypeRegistryImpl typeRegistry) {
     }
 
     public ModuleManagerImpl(Config config, List<Class<?>> classesOnClasspathsToAddToEngine, TypeRegistryImpl typeRegistry) {
```

Comment:
Any parcitular reason why you still need the typeregistry when you've removed it from the class?

Reply:
I must've missed this constructor overload. Thanks for catching it!